Backend/starter_project_mavericks/BlogApp/BlogApp.Domain/Review.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace BlogApp.Domain;
+
+public class Review
```

Comment:
You can inherit Id from BaseDomainEntity

Reply:
I have updated the Review entity so that it can inherit the Id from the BaseDomainEntity